cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -120,10 +123,9 @@ runTxSubmit txFp network = do
   signedTx <- firstExceptT ShelleyTxReadSignedTxError . newExceptT $ readTxSigned txFp
   result   <- liftIO $ submitTx network sktFp signedTx
   case result of
-    TxSubmitSuccess            -> return ()
-    --TODO: use the Cardano.Api.TxSubmit.ErrorRender here
-    TxSubmitFailureByron   err -> liftIO $ print err
-    TxSubmitFailureShelley err -> liftIO $ print err
+    TxSubmitSuccess          -> return ()
+    TxSubmitFailureShelley _ -> left (ShelleyTxSubmitError result)
+    TxSubmitFailureByron   _ -> left (ShelleyTxSubmitError result)
```

Comment:
Is it not odd that we're able to submit Byron transactions from this command?

Reply:
This should get sorted out as we flesh out the typed API